### ISSUE TITLE
Get the server hosted

### ISSUE THREAD

**Issue by seanoshea** [gffny/rgbycch#5]
Would be a good idea to get the server hosted somewhere (S3???). I've literally zero experience of this, but would be happy to give it a try.

**Comment by gffny** [comment]
100% agree. It should be easy enough to set up. I have a bit of experience if you run into any trouble. I have an account that has all of my billing set up. I assume you're talking about the REST API and not the Xcode server for CI?

**Comment by gffny** [comment]
I was thinking of using Docker (http://www.docker.com) to spin up new server instances, which may be worth looking at as well

**Comment by seanoshea** [comment]
Yep - talking about the REST API. The CI stuff should all be handled by travis-ci.

**Comment by seanoshea** [comment]
Yep - been hearing a lot about docker alright. Probably not a bad idea to get that in a separate issue if you think its a good idea.

**Comment by gffny** [comment]
I agree; docker can be a nice to have a little later down the road
